service_mesh/service_mesh_install/ossm-tutorial-kiali.adoc

```diff
@@ -0,0 +1,27 @@
+[id="ossm-kiali-tutorial"]
+= Kiali tutorial
+include::modules/ossm-document-attributes.adoc[]
+:context: ossm-kiali-tutorial
+
+toc::[]
+
+// The following include statements pull in the module files that comprise the assembly. Include any combination of concept, procedure, or reference modules required to cover the user story. You can also include other assemblies.
+
+
+include::modules/ossm-tutorial-kiali-overview.adoc[leveloffset=+2]
+
+include::modules/ossm-tutorial-kiali-accessing-console.adoc[leveloffset=+2]
```

Comment:
I feel like you could use a bit of text in an H2 section before the overview page module to introduce the idea of reviewing all the dashboards in the UI. The rest of the modules should be nested under it, so their nesting would be fine.